BUILD

```
@@ -25,6 +27,18 @@ cc_library(
   ],
 )
 
+cc_library(
+  name = "apiclient",
+  srcs = [
+    "apiclient.cpp",
+  ],
+  hdrs = [
+    "apiclient.h",
+  ],
+  deps = [
+    ":cpprpg_cc_proto",
+  ],
+)
```

Missing empty line between cc_library and cc_test.
```suggestion
)

```